miso-web/src/spring-test/java/uk/ac/bbsrc/tgac/miso/webapp/springtest/ArrayModelControllerST.java

```diff
@@ -0,0 +1,50 @@
+package uk.ac.bbsrc.tgac.miso.webapp.springtest;
+
+import org.junit.Test;
+
+import uk.ac.bbsrc.tgac.miso.core.data.ArrayModel;
+import uk.ac.bbsrc.tgac.miso.core.util.MapBuilder;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class ArrayModelControllerST extends AbstractST {
+
+  private static final String CONTROLLER_BASE = "/arraymodel";
+  private static final Class<ArrayModel> entityClass = ArrayModel.class;
+
+  private List<HashMap<String, Object>> makeObjectMaps(List<Integer> ids) {
+    List<HashMap<String, Object>> mappedProperties = new ArrayList<HashMap<String, Object>>();
```

Comment:
Declare and return as `Map` instead of `HashMap`. Then you don't need to cast below. Always best practice to declare the interface unless you really need something implementation-specific. In this case, this all breaks if `MapBuilder` changes at some point to use a different type of `Map`.